### PR TITLE
fix(k8s): watch_endpoint_slices_schema should be watch_endpoint_slices

### PR DESCRIPTION
**Pull request by paradin**
config watch_endpoint_slices  not watch_endpoint_slices_schema

### Description

Fixes # (issue)

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Comment by monkeyDluffy6017** [comment]
Can we add test cases to cover this?

**Comment by nitishfy** [comment]
@paradin are you still available to work on this?

File: apisix/discovery/kubernetes/init.lua [hunk 1]
```diff
@@ -431,7 +431,7 @@ local function single_mode_init(conf)
 
     local default_weight = conf.default_weight
     local endpoints_informer, err
-    if conf.watch_endpoint_slices_schema then
+    if conf.watch_endpoint_slices then
```
Comment (nitishfy) [inline apisix/discovery/kubernetes/init.lua:434]:
```suggestion
    if conf.watch_endpoint_slices then
```

I don't understand why would you need that? What is the problem it is trying to solve here?

**Review by nitishfy — COMMENTED** [review]
There is no issue linked to this PR and IMO it's not solving any problem so far. I'm not sure why would you want to change the name? Can you help me here?

**Pull request closed without merging**